syntax = "proto3";

package swarm;

import "simulation/swarm/proto/missile_config.proto";
import "simulation/swarm/proto/target_config.proto";

// Simulator configuration.
message SimulatorConfig {
  // Missile configurations.
  repeated MissileConfig missile_configs = 1;

  // Target configurations.
  repeated TargetConfig target_configs = 2;
}
